xrviz/style.py

```diff
@@ -0,0 +1,42 @@
+import panel as pn
+from holoviews.plotting import list_cmaps
+from .sigslot import SigSlot
+
+
+class Style(SigSlot):
+
+    def __init__(self):
+        super().__init__()
+        self.height = pn.widgets.IntSlider(name='height', value=300, start=100, end=1200)
+        self.width = pn.widgets.IntSlider(name='width', value=700, start=100, end=1200)
+        self.cmap = pn.widgets.Select(name='cmap', value='Inferno',
+                                      options=list_cmaps(provider='bokeh', reverse = False))
+        self.colorbar = pn.widgets.Checkbox(name='colorbar', value=True)
+        # colormap_limits
+        self.lower_limit = pn.widgets.TextInput(name='cmap lower limit', width=140)
+        self.upper_limit = pn.widgets.TextInput(name='cmap upper limit', width=140)
+        self.use_all_data = pn.widgets.Checkbox(name='compute min/max from all data', value=True)
```

Comment:
To be revisited later: I think right now this is actually 10/90%ile of the whole dataset if True, or min/max of first slice. I expect users to have opinions about what options should be presented here.

Reply:
Yes, we will change this according to common opinion of users.